mithril-common/src/messages/epoch_settings.rs

```diff
@@ -68,10 +75,28 @@ mod tests {
             "verification_key_signature":"signature_456",
             "operational_certificate":"certificate_456",
             "kes_period":45
-        }]
+        }],
+        "current_cardano_transactions_signing_config": {
+            "security_parameter": 70,
+            "step": 20
+        }
         
         }"#;
 
+    #[derive(Clone, Debug, PartialEq, Default, Serialize, Deserialize)]
+    pub struct EpochSettingsMessageLegacyVersion {
```

Comment:
I feel a bit confused with all these `Legacy`, `Previous` structs, can we structure that a bit more?

Reply:
That seems clear to me, but I can try to find another more sustainable solution.

Reply:
Could you elaborate about the difference between `Legacy` and `Previous`?

Reply:
Rename to:
`EpochSettingsMessageLegacyVersion` => `EpochSettingsMessageUntil**LAST_OPEN_API_VERSION_SUPPORTED**`
`EpochSettingsMessagePreviousVersion` => `EpochSettingsMessageUntil**LAST_OPEN_API_VERSION_SUPPORTED**`

Reply:
Done ✅